### PR TITLE
[PP] Removed old (non GAPI) preprocessing code

### PR DESCRIPTION
**Pull request by anton-potapov**
Removing the old preprocessing code in favour of GAPI based version.

### CODE REVIEW

File: inference-engine/src/preprocessing/ie_preprocess_data.cpp [hunk 1]
```diff
@@ -814,97 +812,12 @@ void PreProcessData::execute(Blob::Ptr &preprocessedBlob, const PreProcessInfo &
...
-            _tmp2->allocate();
-        }
-        res_out = _tmp2;
-    } else {
-        res_out = preprocessedBlob;
-    }
-
-    {
-        OV_ITT_SCOPED_TASK(itt::domains::IEPreproc, "Resize");
-        resize(res_in, res_out, algorithm);
```
Comment (ilya-lavrenov) [inline inference-engine/src/preprocessing/ie_preprocess_data.cpp:873]: should not we remove `resize`?

Comment (anton-potapov) [reply inference-engine/src/preprocessing/ie_preprocess_data.cpp:873]: removed

File: inference-engine/src/preprocessing/ie_preprocess_gapi.cpp [hunk 2]
```diff
@@ -983,15 +973,10 @@ bool PreprocEngine::preprocessBlob(const BlobTypePtr &inBlob, MemoryBlob::Ptr &o
...
-
-    return true;
 }
 
-bool PreprocEngine::preprocessWithGAPI(const Blob::Ptr &inBlob, Blob::Ptr &outBlob,
+void PreprocEngine::preprocessWithGAPI(const Blob::Ptr &inBlob, Blob::Ptr &outBlob,
         const ResizeAlgorithm& algorithm, ColorFormat in_fmt, bool omp_serial, int batch_size) {
-    if (!useGAPI()) {
-        return false;
-    }
```
Comment (ilyachur) [inline inference-engine/src/preprocessing/ie_preprocess_gapi.cpp:980]: Please double check this method. For example in the line 999 you have the next code `return preprocessBlob(inNV12Blob, outMemoryBlob, algorithm, in_fmt, out_fmt, omp_serial,  batch_size);`

Comment (anton-potapov) [reply inference-engine/src/preprocessing/ie_preprocess_gapi.cpp:980]: `preprocessBlob` was changed to return `void` as well. So `return preprocessBlob(...)` is just short version of 
```c++
preprocessBlob();
return;
```

**Review by ilya-lavrenov — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Pull request merged**